src/core-concepts-in-embedded-application/index.md

```diff
@@ -0,0 +1,8 @@
+# Core Concepts in Embedded Application
+
+This chapter covers Rust’s embedded essentials:  
```

Comment:
I think we should re-order these as follows:

* Bootloader
* Configuration
* Logging
* Alloc
* Async

To me this flows better as it builds from the bottom